game_event/views.py

```diff
@@ -63,7 +65,21 @@ def game_events(request):
         current_date = date.today()
         game_events = game_events.filter(time__gte=current_date)
 
-    return render(request, 'game-events.html', {'game_events': game_events})
+    weather = get_weather_from_api(game_events)
+    print(weather)
+    for game in  game_events:
+        game_events_with_weather.append({
+            'id': game.id,
+            'time': game.time,
+            'level_of_game': game.level_of_game,
+            'min_number_of_players': game.min_number_of_players,
+            'max_number_of_players': game.max_number_of_players,
+            'court': game.court,
+            'ball_game': game.ball_game,
+            'weather': weather[game.id]
+        })
+
+    return render(request, 'game-events.html', {'game_events': game_events_with_weather})
```

Comment:
@ZvikaNaorCohen I think that when I change here `return render(request, 'game-events.html', {'game_events': game_events_with_weather})` the game_events value, the tests that fail now, are dependent on his value also. 
lets take it 1on1 and have a look.

Reply:
@ZvikaNaorCohen Failed tests:
`game_event/test_game_event_views.py::TestGameEventServerResponses::test_shown_min_player_filter FAILED                        
game_event/test_game_event_views.py::TestGameEventServerResponses::test_shown_max_player_filter FAILED                        
game_event/test_game_event_views.py::TestGameEventServerResponses::test_shown_max_level_filter FAILED`

Reply:
I made changes in all of the tests in this file in accordance with @ZvikaNaorCohen changes.